db/migrate/20220914094140_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :AuthorId
+      t.integer :PostId
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- As mentioned above, kindly make sure Foreign key is included with the corresponding index 👍🏽